PlaywrigthUITests/PageObjects/DemoQAWebTablesPage.cs

```diff
@@ -89,16 +95,126 @@ public async Task VerifyTableRowContent(string headerName = "First Name", string
+    }
+    
+    public async Task FillFirstName (string firstName)
+    {
+        if(firstName == null) throw new Exception("First Name field should not be null.");
+        await FirstNameField.FillAsync(firstName);
+        Assert.That(await FirstNameField.GetAttributeAsync("value") == firstName, $"The First Name field was not filled with: '{firstName}'.");
+    }
+    
+    public async Task FillLastName (string lastName)
+    {
+        if(lastName == null) throw new Exception("Last Name field should not be null.");
+        await LastNameField.FillAsync(lastName);
+        Assert.That(await LastNameField.GetAttributeAsync("value") == lastName, $"The Last Name field was not filled with: '{lastName}'.");
+    } 
+    
+    public async Task FillEmail (string email)
+    {
+        if(email == null) throw new Exception("Email field should not be null.");
+        await EmailField.FillAsync(email);
+        Assert.That(await EmailField.GetAttributeAsync("value") == email, $"The Email field was not filled with: '{email}'.");
+    }
+    
+    public async Task FillAge (string age)
+    {
+        if(age == null) throw new Exception("Age field should not be null.");
+        await AgeField.FillAsync(age);
+        Assert.That(await AgeField.GetAttributeAsync("value") == age, $"The Age field was not filled with: '{age}'.");
+    }
+    
+    public async Task FillSalary (string salary)
+    {
+        if(salary == null) throw new Exception("Salary field should not be null.");
+        await SalaryField.FillAsync(salary);
+        Assert.That(await SalaryField.GetAttributeAsync("value") == salary, $"The Salary field was not filled with: '{salary}'.");
+    }
+    
+    public async Task FillDepartment (string department)
+    {
+        if(department == null) throw new Exception("Department field should not be null.");
+        await DepartmentField.FillAsync(department);
+        Assert.That(await DepartmentField.GetAttributeAsync("value") == department, $"The Department field was not filled with: '{department}'.");
+    }
+
+    public async Task ClickSubmit()
+    {
+        var submit = Page.GetByRole(AriaRole.Button, new() { Name = "Submit" });
+        await submit.ClickAsync();
+    }
+    
+    public async Task<bool> FindRowByEmail(string email)
+    {
+
+        if (string.IsNullOrWhiteSpace(email)) throw new Exception("Email field is empty.");
+
+        var rows = await Page.Locator("//div[@class=\"rt-tr-group\"]").AllAsync();
+        if (rows == null || rows.Count == 0) throw new Exception("No rows found in the table.");
+
+        bool rowFound = false;
+
+        foreach (var row in rows)
+        {
+            var rowText = await row.InnerTextAsync();
+
+            if (rowText.Contains(email))
+            {
+                rowFound = true;
+                break;
+            }
+        }
+        return rowFound; 
+    }
+    
+    public async Task<string> FindRowContentByEmail(string email, string rowContent = "")
```

Comment:
, string rowContent = "")
should not be parameter, as it is not used as input values
put it as first line inside functions as
var rowContent = "";

Reply:
fixed